recipe/meta.yaml

```diff
@@ -55,4 +53,3 @@ extra:
     - pelson
     - sigmavirus24
     - ocefpaf
-    - mingwandroid
```

Comment:
We might want to make a list of feedstocks that need this change and update them